README.md

```diff
@@ -170,16 +169,6 @@ python
 
 **Note:** If you're unable to download the API specification file because you're using an instance of Looker that is self-signed and errors are thrown, you can explicitly turn off SSL verification by putting `verify_ssl=false` in the `looker.ini` file configuration section.
 
-### Using the Legacy generator
```

Comment:
We should keep this section in because the legacy generator can be used to generate language SDKs we don't have.

Reply:
I added the section back, but omitted the commentary around which SDKs are "currently" available and around changing the API versions (the looker-sample.ini file still explains the apiVersions property with a comment in that file)